index.js

```diff
@@ -31,22 +31,33 @@ module.exports = (config) => {
   // override to consume 'value'
   client.increment = (stat, value, sampleRate, tags) => {
     client.update_stats(stat, value || 1, sampleRate, tags);
-  }
+  };
 
   client.start = () => {
     var startTime = Date.now();
     return {
-      tick: (stat, num) => {
+      tick: (stat, num, _sampleRate, _tags) => {
         const count = num || 1;
+        const sampleRate = _sampleRate || 1;
+        const tags = _tags || [];
```

Comment:
@tik-son It seems `null` or `undefined` is better for the default value of 'tags'.